Source/relay/TourGuide/Quests/8-bit Realm.ash

```diff
@@ -239,7 +240,9 @@ void Q8bitRealmGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [
         
         if (base_quest_state.state_int["currentScore"] < 10000) 
         {
-            keyCompletionSubentry.entries.listAppend("If you max your bonus, you'll have your key in "+pluralise((10000-round(base_quest_state.state_int["currentScore"]))/400," more turn","more turns"));
+            int pointsLeft = 10000 - base_quest_state.state_int["currentScore"];
```

Comment:
whenever possible i was trying to do 1-liners but this one was too long i think. good change to split it up.